runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2337,6 +2336,17 @@ MM_CopyForwardScheme::scanContinuationObjectSlots(MM_EnvironmentVLHGC *env, MM_A
 	}
 }
 
+MMINLINE void
+MM_CopyForwardScheme::scanContinuationObjectSlots(MM_EnvironmentVLHGC *env, MM_AllocationContextTarok *reservingContext, J9Object *objectPtr, ScanReason reason)
```

Comment:
I understand the name of inferred from scanMixedObjectSlots (and I don't know why it's the only one it does not call it simply scanMixedObject), but I'd still rename this method to scanContinuationObject, so it matched other continuation methods.

At some point we can rename scanMixedObjectSlots to scanMixedObject, but out of scope of this change.